examples/web/package.json

```
@@ -1,10 +1,12 @@
 {
   "name": "web-examples",
   "private": true,
-  "version": "0.23.0",
+  "version": "0.26.0",
   "description": "Example of using web plugins in browser",
   "main": "index.js",
   "scripts": {
+    "docker:start": "cd ./docker && docker-compose down && docker-compose up",
+    "docker:startd": "cd ./docker && docker-compose down && docker-compose up -d",
```

```suggestion
    "docker": "cd ./docker && docker-compose down && docker-compose up",
```

`npm run docker -- -d` might be good enough